bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -229,9 +229,6 @@ def __init__(self):
                 self.device_name = "/device:GPU:0"
             else:
                 self.device_name = "/device:CPU:0"
-                num_threads = int(os.environ["BENTOML_NUM_THREAD"])
-                tf.config.threading.set_inter_op_parallelism_threads(num_threads)
-                tf.config.threading.set_intra_op_parallelism_threads(num_threads)
```

Comment:
@parano Because resource strategy already setup `TF_NUM_INTEROP_THREADS` etc., we don't need these lines at all